src/librustc_parse/validate_attr.rs

```diff
@@ -45,9 +48,7 @@ pub fn parse_meta<'a>(sess: &'a ParseSess, attr: &Attribute) -> PResult<'a, Meta
                 }
             }
         },
-        AttrKind::DocComment(comment) => {
-            mk_name_value_item_str(Ident::new(sym::doc, attr.span), comment, attr.span)
-        }
+        AttrKind::DocComment(..) => panic!("unexpected doc comment"),
```

Comment:
`attr` has a `Span`, right? It'd be useful to use `span_bug!()` instead if that is the case, to make things easier if we ever trigger this.

Reply:
`span_bug` is in librustc (it uses `tcx`), so it cannot be used in crates that do not depend on librustc.

Reply:
Any reason not to use `let item = attr.get_normal_item();` here btw?

Reply:
You are right! Updated.